docs/metrics.md

```diff
@@ -26,27 +28,27 @@ Below is the generated list of Prometheus metrics for Hookshot.
 | hookshot_webhooks_http_request | Number of requests made to the hookshot webhooks handler | path, method |
 | hookshot_provisioning_http_request | Number of requests made to the hookshot webhooks handler | path, method |
 | hookshot_queue_event_pushes | Number of events pushed through the queue | event |
-| hookshot_connection_event_failed | The number of events that failed to process | event, connectionId |
-| hookshot_connections | The number of active hookshot connections | service |
+| hookshot_connection_event_failed | Number of events that failed to process | event, connectionId |
+| hookshot_connections | Number of active hookshot connections | service |
 | hookshot_notifications_push | Number of notifications pushed | service |
-| hookshot_notifications_service_up | Is the notification service up or down | service |
+| hookshot_notifications_service_up | Whether the notification service is up or down | service |
 | hookshot_notifications_watchers | Number of notifications watchers running | service |
-| hookshot_feeds_count | The number of RSS feeds that hookshot is subscribed to |  |
-| hookshot_feeds_fetch_ms | The time taken for hookshot to fetch all feeds |  |
-| hookshot_feeds_failing | The number of RSS feeds that hookshot is failing to read | reason |
+| hookshot_feeds_count | Number of RSS feeds that hookshot is subscribed to |  |
```

Comment:
These need editing in their appropriate source files as this list is generated via script. I can take that on unless you feel confident in doing so :)

Reply:
Oh, please do that, it's probably much quicker than if you tell me where I find the right file. :D 

Thanks!